### ISSUE TITLE
Marking a task as completed crashes the kernel

### ISSUE THREAD

**Issue by buti1021** [allegroai/clearml#830]
## Describe the bug
Running mark_completed on a task in a notebook results in the kernel crashing.

## To reproduce

from clearml import Task
task = Task.init()
task.mark_completed()

## Expected behaviour
The kernel should not crash.

## Environment
* Server type app.clear.ml
* ClearML SDK Version 1.8.1
* Python Version 3.9.4
* OS (Windows \ Linux \ Macos) Windows

**Comment by qwaxys** [comment]
Did you install clearml with pip? Did you use `import clearml` ?

**Comment by buti1021** [comment]
Yes, kinda I used from clearml import Task, as shown in the example

**Comment by qwaxys** [comment]
I can reproduce this in Colab. We'll look into this.

**Comment by qwaxys** [comment]
task.mark_completed() should never be used on the Main Task, if you want to close it you should use task.close()
We'll add a warning for this.

If you found this in an example we wrote, could you link it here?

**Comment by buti1021** [comment]
Yeah I got that after the Kernel crashed, I didn't find task.close() before in the documentation, I think it would be a good addition to the existing notebook docs/examples.

**Comment by eugen-ajechiloae-clearml** [comment]
@buti1021 You can't use `mark_*` because that will just modify the status of task on the backend, but the task object itself may still run in the background. This results in an inconsistent state.
We will change the behavior so people will not stumble upon this problem again: we will not allow the user to call `mark_*` functions without setting a `force` parameter to `True` and inform the user that `Task.close()` should be called before.
We will update this thread once these changes are released.
